From a20a01e661b8809259dd80ab55f43e405a99d22d Mon Sep 17 00:00:00 2001
From: zDylant <zdylant@proton.me>
Date: Fri, 18 Nov 2022 09:58:49 -0500
Subject: [PATCH] Modifying default config file to add Kanagawa color scheme.

---
 config.def.h | 54 ++++++++++++++++++++++++++--------------------------
 1 file changed, 27 insertions(+), 27 deletions(-)

diff --git a/config.def.h b/config.def.h
index 91ab8ca..71ad33a 100644
--- a/config.def.h
+++ b/config.def.h
@@ -95,33 +95,33 @@ unsigned int tabspaces = 8;
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
-	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
-	"gray90", /* default foreground colour */
-	"black", /* default background colour */
+    /* 8 normal colors */
+    "#2A2A37",
+    "#E82424",
+    "#76946A",
+    "#FF9E3B",
+    "#7E9CD8",
+    "#957FB8",
+    "#7FB4CA",
+    "#717C7C",
+
+    /* 8 bright colors */
+    "#363646",
+    "#FF5D62",
+    "#98BB6C",
+    "#E6C384",
+    "#A3D4D5",
+    "#9CABCA",
+    "#A3D4D5",
+    "#DCD7BA",
+
+    [255] = 0,
+
+    /* more colors can be added after 255 to use with DefaultXX */
+    "#cccccc",
+    "#555555",
+    "#DCD7BA", /* default foreground colour */
+    "#1F1F28", /* default background colour */
 };
 
 
-- 
2.38.1

